tariff-logic/taric-footwear.test.js

```diff
@@ -15,674 +15,1000 @@ test.each([
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'women' }
+    ],
+    '6403993800'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'unisex/other' }
+    ],
+    '6403993300'
+  ],
+  // Vamp = no
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' }
+    ],
+    { code: '640399', questionKey: 'slippers' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'yes' }
+    ],
+    '6403995000'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' }
+    ],
+    { code: '6403xx', questionKey: 'winterSports' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'yes' }
+    ],
+    { code: '6403xx', questionKey: 'skiBoots' }
+  ],
+  // winterSports = yes
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'yes' },
+      { questionKey: 'skiBoots', answerKey: 'skiBoots' }
+    ],
+    '6403120000'
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'yes' },
+      { questionKey: 'skiBoots', answerKey: 'other' }
+    ],
+    '6403190000'
+  ],
+  // winterSports = no
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' }
+    ],
+    { code: '640399', questionKey: 'sports' }
+  ],
+  [
+    '6403xx',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'winterSports', answerKey: 'no' },
+      { questionKey: 'sports', answerKey: 'no' }
+    ],
+    { code: '640399', questionKey: 'lengthOfInsole' }
```

Comment:
Lack same test case but sports => yes (should also return lengthOfInsole question)

Reply:
sports = yes resulting in the question lengthOfInsole